admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/SchemaServiceTest.java

```diff
@@ -306,9 +306,8 @@ public void testPublishIdentitySchemaWithException() throws Exception {
 	
 	@Test
 	@WithUserDetails("global-admin")
-	public void testDeleteIdentitySchema() throws Exception {		
-		Mockito.when(identitySchemaRepository.deleteIdentitySchema(Mockito.anyString(),  Mockito.any(LocalDateTime.class), 
-				Mockito.anyString())).thenReturn(1);		
+	public void testDeleteIdentitySchema() throws Exception {
+		Mockito.when(identitySchemaRepository.findIdentitySchemaById(Mockito.anyString())).thenReturn(draftSchema);
 		identitySchemaService.deleteSchema("test-test");
```

Comment:
There is no assertion nor exception that is caught in the test method.

Reply:
taken care